module_auto_update/data/cron_data.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record model="ir.cron" id="module_check_upgrades_cron">
+            <field name="name">Check for Module Upgrades</field>
+            <field name="active" eval="True"/>
+            <field name="user_id" ref="base.user_root"/>
+            <field name="interval_number">1</field>
+            <field name="interval_type">days</field>
+            <field name="numbercall">-1</field>
+            <field name="nextcall" eval="(DateTime.now() + timedelta(days= +1)).strftime('%Y-%m-%d 3:00:00')"/>
+            <field name="model" eval="'ir.module.module'"/>
+            <field name="function" eval="'check_upgrade_availability'"/>
```

Comment:
Simple strings should just be in the XML text node. Eval is for python code:

`<function>check_upgrade_availability</function>`